docs/home/contribute/localization.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Localizing Kubernetes Documentation
+---
+
+We're happy to add localizations of Kubernetes documentation to the website!
+
+Localizations must meet the following requirements for _workflow_ (how to localize) and _output_ (what to localize).
+
+## Workflow  
+
+- All localization work must happen within a Kubernetes organization repository specifically dedicated to localization.
+
+    For example: https://github.com/kubernetes/kubernetes-docs-cn
+
+    To open a localization repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+
+- Teams must track their overall progress with a GitHub project.
```

Comment:
Should we make it clear that contributors should fix the upstream issues which they find in l10n work or not?

Reply:
There's a blurb below about limiting upstream fixes to a maximum of 1 squashed PR per week, but we can change the language to more specifically welcome upstream fixes within that limitation. 👍